### PR TITLE
Ensure player is set at start location if there is one

### PR DESCRIPTION
**Pull request by StraToN**
This is done even if there is no ESC script attached to the room. 
First default position will then be the start ESCLocation if there is one.
Else, fallback to origin (0,0).

Fixes godot-escoria/escoria-issues#39

### CODE REVIEW

File: addons/escoria-core/game/core-scripts/esc_room.gd [hunk 1]
```diff
@@ -82,8 +82,8 @@ func _ready():
 		add_child(game)
 		move_child(game, 0)
 	
-	if escoria.main.current_scene == null:
-		escoria.main.set_scene(self)
+#	if escoria.main.current_scene == null:
```
Comment (dploeger) [inline addons/escoria-core/game/core-scripts/esc_room.gd:85]: Remove this commented out code.

File: addons/escoria-core/game/core-scripts/esc_room.gd [hunk 2]
```diff
@@ -113,22 +113,24 @@ func _ready():
 	else:
 		is_run_directly = true
 	
+	# Manage player location at room start
+#	if (escoria.globals_manager.get_global("ESC_LAST_SCENE") == null \
+#			or escoria.globals_manager.get_global("ESC_LAST_SCENE").empty()) \
```
Comment (dploeger) [inline addons/escoria-core/game/core-scripts/esc_room.gd:118]: Remove this commented out code.

File: game/rooms/room15/esc/right_exit.esc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:exit_scene
+#set_sound_state _sound res://game/sfx/sounds/doorOpen_2.ogg false
```
Comment (dploeger) [inline game/rooms/room15/esc/right_exit.esc:2]: This file can be removed when there's only commented out code.

File: game/rooms/room15/esc/room15_unused.esc [hunk 4]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (dploeger) [inline game/rooms/room15/esc/room15_unused.esc:1]: Why is this called room15_unused?

**Review by dploeger — CHANGES_REQUESTED** [review]
Just some smaller cosmetics.

File: addons/escoria-core/game/core-scripts/esc_room.gd (hunk 1 above)
Comment (StraToN) [reply addons/escoria-core/game/core-scripts/esc_room.gd:85]: Oops 😅 Done

File: addons/escoria-core/game/core-scripts/esc_room.gd (hunk 2 above)
Comment (StraToN) [reply addons/escoria-core/game/core-scripts/esc_room.gd:118]: Same. Sorry, I did some tests and forgot it :D

File: game/rooms/room15/esc/room15_unused.esc (hunk 4 above)
Comment (StraToN) [reply game/rooms/room15/esc/room15_unused.esc:1]: The intention was to show that the file was not used... but having no file is better :)

File: game/rooms/room15/esc/right_exit.esc (hunk 3 above)
Comment (StraToN) [reply game/rooms/room15/esc/right_exit.esc:2]: Done

**Review by dploeger — APPROVED** [review]
Nice. Thanks a lot. 🎁

**Pull request merged**